workspace-manager-clienttests/src/main/java/scripts/utils/ResourceMaker.java

```diff
@@ -118,7 +120,7 @@ public static GcpGcsBucketResource makeControlledGcsBucketUserShared(
 
     List<GcpGcsBucketLifecycleRule> lifecycleRules = new ArrayList<>(List.of(lifecycleRule));
 
-    String bucketName = ClientTestUtils.getTestBucketName();
+    String bucketName = ClientTestUtils.generateTestResourceName();
```

Comment:
`resourceName`

Reply:
this is actually the bucket name, not the WSM resource name. Renamed `generateTestResourceName` to make it clearer